capsules/src/process_console.rs

```diff
@@ -268,6 +277,7 @@ impl<'a, A: Alarm<'a>, C: ProcessManagementCapability> ProcessConsole<'a, A, C>
         rx_buffer: &'static mut [u8],
         queue_buffer: &'static mut [u8],
         cmd_buffer: &'static mut [u8],
+        cmd_history_buffer: &'static mut [[u8; COMMAND_BUF_LEN]; COMMAND_HISTORY_LEN],
```

Comment:
I do not like the array in array idea, I suggest working with @matrix22 , that submitted https://github.com/UPB-CS-OpenSourceUpstream/tock/pull/16, and adding better data types. Moreover, using generics is a much better idea.